app/src/main/java/org/simple/clinic/summary/PatientSummaryUpdate.kt

```diff
@@ -62,6 +62,17 @@ class PatientSummaryUpdate : Update<PatientSummaryModel, PatientSummaryEvent, Pa
     )
   }
 
+  private fun currentFacilityLoaded(model: PatientSummaryModel, event: CurrentFacilityLoaded): Next<PatientSummaryModel, PatientSummaryEffect> {
+    if (event.facility.config.teleconsultationEnabled == true) {
+      return next(
+          model.currentFacilityLoaded(event.facility),
+          FetchFacilityTeleconsultationInfo(event.facility.uuid)
+      )
+    }
+
+    return next(model.currentFacilityLoaded(event.facility))
```

Comment:
I took a stab at simplifying this:

```kotlin
val updatedModel = model.currentFacilityLoaded(event.facility)
    
return if(updatedModel.isTeleconsultationEnabled) {
      next(updatedModel)
} else {
      next(updatedModel)
}
```

Reply:
OMG, I completely forgot about extracting the model 🤦

Reply:
Also, I made a mistake in my own suggestion by not adding the effect to the `if` branch. 😓